setup.cfg

```
@@ -22,11 +22,22 @@ ignore =
 	# (check https://github.com/wemake-services/wemake-python-styleguide/issues/1441)
 	WPS436,
 	# Our private objects are fine to import
-	WPS450 
+	WPS450,
+	# We need multine loops
+	WPS352,
```

Please order the ignored errors alphabetically.